website/server/controllers/api-v3/hall.js

```diff
@@ -310,16 +322,63 @@ api.updateHero = {
       }
     }
 
+    if (updateData.changeApiToken) hero.apiToken = common.uuid();
+
     const savedHero = await hero.save();
     const heroJSON = savedHero.toJSON();
     heroJSON.secret = savedHero.getSecretData();
     const responseHero = { _id: heroJSON._id }; // only respond with important fields
-    heroAdminFields.split(' ').forEach(field => {
+    heroAdminFieldsToShow.split(' ').forEach(field => {
       _.set(responseHero, field, _.get(heroJSON, field));
     });
 
     res.respond(200, responseHero);
   },
 };
 
+/**
+ * @api {get} /api/v3/hall/heroes/party/:groupId Get any Party given its ID
+ * @apiParam (Path) {UUID} groupId party's group ID
+ * @apiName GetHeroParty
+ * @apiGroup Hall
+ * @apiPermission Admin
+ *
+ * @apiDescription Returns some basic information about a given Party,
+ # to assist admins with user support.
+ *
+ * @apiSuccess {Object} data The party object (contains computed fields
+ * that are not in the Group model)
+ *
+ * @apiUse NoAuthHeaders
+ * @apiUse NoAccount
+ * @apiUse NoUser
+ * @apiUse NotAdmin
+ * @apiUse groupIdRequired
+ * @apiUse GroupNotFound
+ */
+api.getHeroParty = { // @TODO XXX add tests
+  method: 'GET',
+  url: '/hall/heroes/party/:groupId',
+  middlewares: [authWithHeaders(), ensureAdmin],
```

Comment:
so for the routes here, we're adding the new privileges in the user schema but not using them? I mean all the routes here are checking contributor.admin, not one of the new privileges so they're still accessible from the api